ReactAndroid/build.gradle

```diff
@@ -135,7 +119,8 @@ task prepareGlog(dependsOn: dependenciesPath ? [] : [downloadGlog], type: Copy)
 }
 
 task downloadJSCHeaders(type: Download) {
-    def jscAPIBaseURL = 'https://svn.webkit.org/repository/webkit/!svn/bc/174650/trunk/Source/JavaScriptCore/API/'
+    // in sync with webkit SVN revision 174650
+    def jscAPIBaseURL = 'https://raw.githubusercontent.com/WebKit/webkit/38b15a3ba3c1b0798f2036f7cea36ffdc096202e/Source/JavaScriptCore/API/'
```

Comment:
It feels more reliable to use only github for 3rd party code downloads.
But I am not 100% sure if raw.githubusercontent.com rate limits are worse or bettern than svn.webkit.org